### DISCUSSION TITLE
picture-kaleidoscope

### DISCUSSION THREAD

**Discussion by noideaforaccountname** [godot-nim/godot#131] [Show and tell]
Hi,
godot-nim is fun. thank you!
ive done a little demo for drawing with draw. loads a picture and draws a kaleidoscope from random picturerects.
[kaleidoscope.zip](https://github.com/user-attachments/files/18062343/kaleidoscope.zip)

**Comment by panno8M** [comment]
Hello! Thanks for using gdext-nim and making a work!

I want to post this in the demo, which way do you prefer?

1. PR the project to the godot-nim/demo repository
1. Post a link to your project in godot-nim/demo/readme.md
1. Both 1 and 2
1. Don't want it used for demonstrations.

Btw I'd recommend 3 :)

Cheers!

**Reply by noideaforaccountname** [reply]
thx for answer. whatever you want - if you recommend 3 then 3. btw: the examplepicture was made by an ai, so there are no problems with rights. or you change it to an other picture if you like.

**Reply by panno8M** [reply]
Nice. I would like you to PR (Pull Request) me if you can, because it will leave information about who contributed.
I want you to fork godot-nim/demo and create a picture-kaleidoscope directory there and make a pull request. Do you know how to do it?

**Reply by noideaforaccountname** [reply]
ive never done this, but i tried please have a look...

**Comment by noideaforaccountname** [comment]
i changed and deleted the dlls, ok? if it works, we can delete this thread?

**Reply by panno8M** [reply]
It's fine, but I want it to stay. I think this precedent will make it easier for others to follow.
In that sense I am quite grateful to you; I forgot to mention ;)
